src/tests/ie_test_utils/common_test_utils/data_utils.hpp

```diff
@@ -253,6 +254,77 @@ void inline fill_random_unique_sequence(T* rawBlobDataPtr,
     std::copy(elems.begin(), elems.end(), rawBlobDataPtr);
 }
 
+/** @brief Fill tensor with random data.
```

Comment:
Why do you add a lot of new generators? hat is the difference with old one? Do you really need to use them?

Reply:
Yeah, as I can see, there is no any `fill_ *` functions for ov::tensors (there is create_and_fill(), but I need to fill existing tensors). That is why I added them